eng/common/templates/job/job.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         env:
           TeamName: $(_TeamName)
         continueOnError: ${{ parameters.continueOnError }}
-        condition: and(succeeded(), in(variables['_SignType'], 'real', 'test'), eq(variables['Agent.Os'], 'Windows_NT'))
+        condition: and(succeeded(), in(variables['_SignType'], 'real', 'test'), eq(variables['Agent.Os'], 'win'))
```

Comment:
Has the `Agent.Os` value actually changed? Isn't this something coming from AzDO?

Reply:
Yeah, this is wrong. Anyway, this file shouldn't be changed as it is an arcade file